src/base/global.h

```diff
@@ -26,5 +26,20 @@
  * exception statement from your version.
  */
 
+#pragma once
+
 const char C_TORRENT_FILE_EXTENSION[] = ".torrent";
 
+
+#if QT_VERSION < QT_VERSION_CHECK(5, 7, 0)
+template <typename T>
+constexpr typename std::add_const<T>::type &qAsConst(T &t) noexcept { return t; }
```

Comment:
`#include <type_traits>` ?

Reply:
Oops... we need `#include <QtGlobal>` for QT_VERSION. It also includes type_traits.
Coincidentally, this header always follows after some Qt header and we don't have compile errors.